docs/IE_DG/supported_plugins/GNA.md

```diff
@@ -159,6 +162,29 @@ Heterogeneous plugin was tested with the Intel&reg; GNA as a primary device and
 
 > **NOTE:** Due to limitation of the Intel&reg; GNA backend library, heterogenous support is limited to cases where in the resulted sliced graph, only one subgraph is scheduled to run on GNA\_HW or GNA\_SW devices.
 
+## Recovery from interruption by high-priority Windows audio processes\*
+
+As noted in the introduction, GNA is designed for real-time workloads such as noise reduction or speech recognition. 
```

Comment:
I would remove "speech recognition" since there is no audio glitch problem in that application.

Reply:
Fixed